data/src/main/scala/org/apache/predictionio/data/store/LEventStore.scala

```diff
@@ -20,15 +20,27 @@ package org.apache.predictionio.data.store
 
 import org.apache.predictionio.data.storage.Storage
 import org.apache.predictionio.data.storage.Event
-
 import org.joda.time.DateTime
 
-import scala.concurrent.Await
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.concurrent.duration.Duration
 
 /** This object provides a set of operation to access Event Store
-  * without going through Spark's parallelization
+  * without going through Spark's parallelization.
+  *
+  * Note that blocking methods of this object uses
+  * `scala.concurrent.ExecutionContext.Implicits.global` internally.
+  * Since this is a thread pool which has a number of threads equal to available
+  * processors, parallelism is limited up to the number of processors.
+  *
+  * If this limitation become bottleneck of resource usage, you can increase the
+  * number of threads by declaring following VM options before calling "pio deploy":
+  *
+  * <pre>
+  * export JAVA_OPTS="$JAVA_OPTS \
+  *   -Dscala.concurrent.context.numThreads=1000 \
+  *   -Dscala.concurrent.context.maxThreads=1000"
+  * </pre>
```

Comment:
@dszeto Is this comment understandable? Since I'm not confident that it explains correctly and clearly, I'd like to request your confirmation.

Reply:
@Wei-1 Thanks for your feedback! I added line comment on importing as below:
```scala
// Import here to ensure ExecutionContext.Implicits.global is used only this method
import scala.concurrent.ExecutionContext.Implicits.global
```

Reply:
btw, only `in` this method?

Reply:
LGTM. We can link to https://docs.scala-lang.org/overviews/core/futures.html#the-global-execution-context for a detail explanation if you'd like to.